src/home/views.py

```diff
@@ -25,9 +24,8 @@ def get_context_data(self, **kwargs):
 
         aids_qs = Aid.objects.open().published()
         financers = aids_qs.values_list('financers', flat=True)
-        instructors = aids_qs.values_list('instructors', flat=True)
         nb_backers = Backer.objects \
```

Comment:
Ca me paraissait sous-optimal de passer par un values_list, donc j'ai creusé et j'ai trouvé ça : 
```
aids_qs = Aid.objects.open().published()
nb_backers = Backer.objects.filter(financed_aids__in=aids_qs).distinct().count()
```

Reply:
Ca s'appelle un reverse lookup, j'ai découvert au passage https://stackoverflow.com/questions/12986010/getting-all-unique-values-stored-in-a-many-to-many-field

Reply:
Intéressant! :) Par contre avec les deux méthodes j'ai exactement le même nombres de requêtes SQL générées (9) dans un temps similaire. ça te paraît sous-optimal en terme de clarté du code?

Reply:
J'ai essayé de mon coté, ca semble plus rapide mais ca ne se joue pas à beaucoup. En tout cas je trouve ça plus beau mais je suis peut-être biaisé ^^
Et je ne sais pas si `aids_qs = Aid.objects.prefetch_related('financers').open().published()` augmenterait la vitesse.
Et also `context['nb_aids'] = aids_qs.count()`

Reply:
Dernière simplification et après j'arrête : remplace `.open().published()` par `.live()` 😆 
@lazybird je te laisse partager ton avis et faire pencher la balance dans un sens ou un autre 🙃

Reply:
Le prefetch related ne change pas la donne selon la DJDT.

Reply:
Complétement okay pour le `.live() `et ok aussi pour l'argument esthétique. 

Par contre avec le `context['nb_aids'] = aids_qs.count()` je me demande si on ne risque pas d'avoir des doublons. J'imagine que c'était dans cette optique qu'il y avait `context['nb_aids'] = aids_qs.values('id').count()` nop?